modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -221,7 +221,9 @@ export default class ScenegraphLayer extends Layer {
         fs: this.addVersionToShader(fs),
         modules,
         isInstanced: true
-      }
+      },
+      // tangents are not supported
+      useTangents: false
```

Comment:
Is this still needed after cutting out all tangent code?

Reply:
Yes.

Reply:
There is some code inside the luma module.